### DISCUSSION TITLE
如何默认收拢combo，并避免嵌套combo

### DISCUSSION THREAD

**Discussion by Shinonome233** [antvis/G6#3375] [Q&A] [answered]
1. graph展示时默认会展开combo，可以设置默认收拢combo吗？
2. 希望可以拖拽combo，但不想这个combo被嵌套到别的combo中，请问有什么办法可以做到吗？

**Comment by Yanyan-Wang** [comment, marked as answer]
1. combo 数据中设置 collapsed: true
2. drag-combo behavior 有配置项，可以去官网文档看看
